config/routes.rb

```diff
@@ -5,7 +5,8 @@
 
   namespace :admin do
     resources :merchants, only: [:index, :show, :update, :edit]
-    resources :invoices, only: [:index, :show, :update]
+    resources :invoices, only: [:index, :show]
```

Comment:
concerned that :update leaves as a route, unless that's okay because the route is covered elsewhere?

Reply:
Did everything work without :update being enabled? If :update isn't used anywhere yet it's probably fine to be left off. If we need the route, it can always be added back when we get to that.